pulp_rpm/app/viewsets.py

```diff
@@ -386,72 +442,9 @@ class UlnRemoteViewSet(RemoteViewSet):
     endpoint_name = "uln"
     queryset = UlnRemote.objects.all()
     serializer_class = UlnRemoteSerializer
-    queryset_filtering_required_permission = "rpm.view_ulnremote"
-
-    DEFAULT_ACCESS_POLICY = {
-        "statements": [
-            {
-                "action": ["list"],
-                "principal": ["authenticated"],
-                "effect": "allow",
-                "condition": "has_model_perms:rpm.view_ulnremote",
-            },
-            {
-                "action": ["retrieve"],
-                "principal": "authenticated",
-                "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.view_ulnremote",
-            },
-            {
-                "action": ["create"],
-                "principal": "authenticated",
-                "effect": "allow",
-                "condition": "has_model_perms:rpm.add_ulnremote",
-            },
-            {
-                "action": ["update", "partial_update"],
-                "principal": "authenticated",
-                "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.change_ulnremote",
-            },
-            {
-                "action": ["destroy"],
-                "principal": "authenticated",
-                "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.delete_ulnremote",
-            },
-            {
-                "action": ["list_roles", "add_role", "remove_role"],
-                "principal": "authenticated",
-                "effect": "allow",
-                "condition": ["has_model_or_obj_perms:rpm.manage_roles_ulnremote"],
-            },
-        ],
-        "creation_hooks": [
-            {
-                "function": "add_roles_for_object_creator",
-                "parameters": {"roles": "rpm.ulnremote_owner"},
-            }
-        ],
-    }
+    queryset_filtering_required_permission = "rpm.view_rpmremote"
```

Comment:
is it desirable to treat url remote as rpm? i could see them having separate permissions and policy as you had it before

Reply:
those remote only differs just because a way of authentication, so they could be still considered as one type.

Reply:
I don't know, @dralley do we want to distinguish between Remote types?

Reply:
I don't think so.